### DISCUSSION TITLE
Can I get the actual size of the scroll indicator in ScrollArea?

### DISCUSSION THREAD

**Discussion by kajakpalec** [mantinedev/mantine#3278] [Q&A]
I want to make something similar to infinite scroll in my app. 
For this I'm using ScrollArea, which has `onScrollPositionChange` prop. This gives me the current position of the scroll indicator, but the problem is that it doesn't include its size. 

Is there a way to get the scroll indicator size?

**Comment by rtivital** [comment]
You can get viewport element ref and get its bounding client rect – see example – https://mantine.dev/core/scroll-area/#scroll-to-position
